frontend/src/app/components/LayoutWrapper.js

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+const LayoutWrapper = ({ children, id = null, flexModifier, helperClass = null }) => {
```

Comment:
(very optional suggestion) The only cases where the `id` is needed are the `Header` and `Footer` components. Maybe just add wrapper divs with `id`s to those components, and remove that option here?